examples/creatives/hover_nonlinear.js

```diff
@@ -0,0 +1,141 @@
+const DIMENSION_GROWTH = 1.4;
+const OFFSET_SHIFT = .2;
+const EDGE_DIMENSION_GROWTH = 1.2;
```

Comment:
What's the difference between dimension growth & edge dimension growth?

Reply:
EDGE_DIMENSION_GROWTH is applied when the iframe is at the edge of the player (ie: when the x/y offset is 0, or when you try and expand the iframe the calculations would cause it to expand outside the player dimensions). When this occurs we have to make sure the %s are pretty much equivalent to those that occur when the iframe is lets say at the center of the player div and there is room to expand without going over the player's borders (in which case DIMENSION_GROWTH would be applied since the OFFSET_SHIFT is capable of being applied.